src/tree/EnvironmentTreeItem.ts

```diff
@@ -124,6 +120,10 @@ export class EnvironmentTreeItem extends AzureParentTreeItem implements IAzureRe
             children.push(this.appSettingsTreeItem, this.functionsTreeItem);
         }
 
+        if (this.inWorkspace) {
```

Comment:
![image](https://user-images.githubusercontent.com/5290572/117226324-3f7b2980-adc9-11eb-9d79-354452726c05.png)

😏 

I think we should just fix the `compareChildrenImpl` command.  If we do something like 

```
if (item1 instanceof GenericTreeItem && !(item2 instanceof GenericTreeItem)) {
            return -1;
        } else if (!(item1 instanceof GenericTreeItem) && item2 instanceof GenericTreeItem) {
            return 1;
        } else {
            return super.compareChildrenImpl(item1, item2);
        }
}
```

Reply:
Our pattern for app service and functions is to sort this level of the tree manually based on what we think is most important for users, not alphabetically (which is what `super.compareChildrenImpl` would do). I think we should keep doing the same here.

I'm guessing you don't want "Actions" at the bottom, which is possible if you sort alphabetically and if/when we ever localize.